src/Components/test/E2ETest/e2eTestSettings.json

```diff
@@ -1,4 +1,11 @@
 {
   "DefaultWaitTimeoutInSeconds": 20,
-  "ScreenShotsPath": "../../screenshots"
+  "ScreenShotsPath": "../../screenshots",
+  "SauceTest":  true,
+  "Sauce": {
+    "Username": "placeholder",
```

Comment:
This and the key will come from env variables.

Reply:
You don't need this. I believe we map environment variables in the config for E2E tests already, so feel free to skip this.